src/home/frontend/components/notification/NotificationBell.js

```diff
@@ -6,8 +6,6 @@ import Menu from "@mui/material/Menu";
 import MenuItem from "@mui/material/MenuItem";
 import Box from "@mui/material/Box";
 import Tooltip from "@mui/material/Tooltip";
-// import BasicMenu from "../menu/BasicMenu";
```

Comment:
never use commented out code. comments should be used only for commenting purpose. 
remove all commented out code and unused code.